site/content/contribute/getting-started/inactive-contributions.md

```diff
@@ -7,15 +7,15 @@ subsection: Getting Started
 
 This process describes how inactive contributions are managed at Mattermost, inspired by the [Kubernetes project](https://github.com/kubernetes/kubernetes):
 
-1. After 14 days of inactivity, a contribution becomes stale.
+1. After 10 days of inactivity, a contribution becomes stale.
  - If action is required from submitter, Community Coordinator asks if the team can help clarify previous feedback or provide guidance on next steps, and adds `lifecycle/1:stale` label to the contribution.
  - If action is required from reviewers, Community Coordinator asks reviewers to share feedback or help answer questions, and adds `lifecycle/1:stale` label to the contribution. The Coordinator will follow up with reviewers until a response is received.
 
-2. After another 14 days of inactivity, a contribution becomes inactive.
+2. After another 4 days of inactivity, a contribution becomes inactive.
```

Comment:
this should stay at 14 days.

Reply:
Please see
 - https://community.mattermost.com/private-core/pl/odr33ttid3f4bgsq1kw8gjznih
 - https://community.mattermost.com/private-core/pl/cqsc8o34hfdkdn7qpe7nxcbrke

for context. If we keep it at 14 days, then the lifecycle will remain about the same.

Reply:
Let's discuss this via DM first.